assets/core/js/scripts.js

```
@@ -23,6 +23,125 @@ function json_parse(data, debug) {
+        for (var key in errors) {
+            if (errors.hasOwnProperty(key)) {
+                var errorMsg = Array.isArray(errors[key]) ? errors[key].join(', ') : errors[key];
+                errorHtml += '<li><strong>' + key.replace(/_/g, ' ').replace(/\b\w/g, l => l.toUpperCase()) + ':</strong> ' + errorMsg + '</li>';
+            }
+        }
+        errorHtml += '</ul></div>';
+        $(targetSelector).html(errorHtml);
+    }
+
+    // Highlight fields with errors
+    for (var key in errors) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'key'. Consider to delete it or rename it.

'key' is defined here:


(lint/suspicious/noRedeclare)